languages/ruby/README.md

```diff
@@ -113,8 +124,8 @@ cd ..
 export ACCESS_TOKEN=""
 export ORGANIZATION_ID=""
 
-export API_URL=https://localhost:8080/api
-export IDENTITY_URL=https://localhost:8080/identity
+export API_URL=http://localhost:4000
+export IDENTITY_URL=http://localhost:33656
```

Comment:
After messing with this for a bit, I got these proxy endpoints working if: 

- You generate a valid SSL Certificate following instructions [here](https://contributing.bitwarden.com/getting-started/clients/web-vault/#ssl-certificate)
- Run `export NODE_EXTRA_CA_CERTS="$(mkcert -CAROOT)/rootCA.pem"`
- Have the web client running to have these proxies running.

I don't have a strong opinion either way, but we probably need more clarity on prerequisites for using the proxy endpoints.